web/src/test/java/org/openmrs/scheduler/web/controller/SchedulerFormControllerTest.java

```diff
@@ -151,6 +151,7 @@ public void onSubmit_shouldNotRescheduleAnExecutingTask() throws Exception {
 		assertTrue(mav.getModel().isEmpty());
 		
 		Assert.assertSame(oldTaskInstance, task.getTaskInstance());
+		super.deleteAllData();
```

Comment:
I think calling deleteAllData() should be enough without the super since it is not overidden in this class